app/controllers/application_controller.rb

```diff
@@ -48,7 +49,8 @@ def after_sign_in_path_for(resource)
     return edit_user_path id: current_user.id if session[:pending_organisation_id]
     return organisation_path(current_user.organisation) if current_user.organisation
     return session[:previous_url] if session[:previous_url]
-    return organisation_path(Organisation.find(current_user.pending_organisation_id)) if current_user.pending_organisation_id
+    return organisation_path(Organisation.find(current_user.pending_organisation_id)) if
```

Comment:
I noted two things:
1. `Organisation.find` is not required
2. `return pending_organisation_path_for_current_user if current_user.pending_organisation_id` itself exceeds limit of 90.